modules/twinklewarn.js

```diff
@@ -1163,15 +1163,18 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 	if (!Twinkle.getPref('oldSelect')) {
 		$('select[name=sub_group]')
 			.chosen({width: '100%', search_contains: true})
-			.change(Twinkle.warn.callback.change_subcategory);
+			.change(Twinkle.warn.callback.change_subcategory)
+			.on('chosen:showing_dropdown', function () {
+				$(this).parents('div').css('overflow', 'visible');
+				$('#twinklewarn-previewbox').css({ 'max-height': '100px', 'overflow': 'hidden' });
+			})
+			.on('chosen:hiding_dropdown', function () {
+				$(this).parents('div').css('overflow', '');
+				$('#twinklewarn-previewbox').css({ 'max-height': '', 'overflow': '' });
```

Comment:
Shouldn't this be `visible`?
```suggestion
				$('#twinklewarn-previewbox').css({ 'max-height': '', 'overflow': 'visible' });
```